### ISSUE TITLE
webxr example is broken with 'no method named `xr` found for struct `Navigator` in the current scope'

### ISSUE THREAD

**Issue by expenses** [rustwasm/wasm-bindgen#2862]
### Describe the Bug

Hi, while trying to run the webxr example with `RUSTFLAGS=--cfg=web_sys_unstable_apis npm run serve` I ran into this issue:
```rust
error[E0599]: no method named `xr` found for struct `Navigator` in the current scope
  --> examples/webxr/src/lib.rs:81:28
   |
81 |         let xr = navigator.xr();
   |                            ^^ method not found in `Navigator`
```

### Steps to Reproduce

1. Checkout wasm-bindgen commit 4caa98165c65984e3eba3dd7825f3bf44955d127
2. cd into `examples/webxr`
3. Run `npm install`
4. Run `RUSTFLAGS=--cfg=web_sys_unstable_apis npm run serve`
5. Get the above error

**Comment by expenses** [comment]
I found a fix; it looks like a `XrSystem` feature flag was not generated. This gets the code to run:

```diff
diff --git a/crates/web-sys/Cargo.toml b/crates/web-sys/Cargo.toml
index 19f804a0..77a01baa 100644
--- a/crates/web-sys/Cargo.toml
+++ b/crates/web-sys/Cargo.toml
@@ -1484,6 +1484,7 @@ XrSessionEventInit = []
 XrSessionInit = []
 XrSessionMode = []
 XrSpace = ["EventTarget"]
+XrSystem = []
 XrTargetRayMode = []
 XrView = []
 XrViewerPose = ["XrPose"]
diff --git a/examples/webxr/Cargo.toml b/examples/webxr/Cargo.toml
index 4127ec5e..1d90d11c 100644
--- a/examples/webxr/Cargo.toml
+++ b/examples/webxr/Cargo.toml
@@ -54,6 +54,7 @@ features = [
   'XrSessionInit',
   'XrSessionMode',
   'XrSpace',
+  "XrSystem",
   'XrTargetRayMode',
   'XrView',
   'XrViewerPose',
```

**Comment by expenses** [comment]
Solved by #2863.
